contracts/mining.js

```diff
@@ -615,7 +615,10 @@ actions.createPool = async (payload) => {
   const params = await api.db.findOne('params', {});
   const { poolCreationFee } = params;
 
-  if (typeof externalMiners === 'string' && !api.assert(callingContractInfo, 'must be called from a contract')) return;
+  if (externalMiners) {
```

Comment:
I think this still isn't quite right.  If someone passes in 0 for externalMiners, then this block of code will never trigger.  To make it completely safe, it should be `if (externalMiners !== undefined) {`

Reply:
Yes I was going through here too quickly. Fixed.